pkg/compiler/compiler_test.go

```diff
@@ -23,14 +23,17 @@ const examplePath = "../../examples"
 const exampleCompilePath = "testdata/compile"
 const exampleSavePath = exampleCompilePath + "/save"
 
```

Comment:
Would require some `_ = versionutil.TestVersion` here probably, but it's a little easier than a full `init()` here.

Reply:
Fixed.